### ISSUE TITLE
Custom keyboard shortcuts?

### ISSUE THREAD

**Issue by Alok** [inducer/pudb#294]
Is there a way to configure this or would it have to be built out (happy to hack on it).

**Comment by inducer** [comment]
No, there's not currently a way to configure this--but I'd be happy to consider a patch.

**Comment by Alok** [comment]
Where is the relevant code in the source?

**Comment by asmeurer** [comment]
Right now the keyboard shortcuts are all hardcoded in debugger.py. I would start by extracting them to a single dictionary in a separate file, so that it can easily be overloaded with configuration (similar to how the theming works, except without any notion of inheritance). Every keyboard shortcut will need a name. I would namespace them based on what view they apply to (source view, var view, global, etc.).
